analysis/analysis.py

```diff
@@ -92,6 +93,38 @@ def run_analysis():
     show_response_groups(response_groups_freq_dicts)
     total_view_time = compute_total_view_time(student_data)
     print(f'The total view time of all students was {total_view_time}.')
+    print(count_revists(student_data))
+
+
+def count_revists(data):
+    """
+    Returns the average number of revisits per question
```

Comment:
please specify 'mean' rather than 'average' -- average can mean a lot of things!

Reply:
In general, the median is a better measure of average on anything where huge outliers may occur anyhow.  (We increase the mean income in the US by giving Zuckerberg more money.  We increase the median only by helping the poorest 50%)

Reply:
Changed to specify 'mean' rather than 'average'. 

@mscuthbert We thought huge outliers would not be an issue in this case because we were counting the number of revisits rather than the actual revisit times. But just to be sure, we implemented a function to compute medians for comparison. The resulting data was not too far off from that of the mean function's.